learningobjects/tests/test_api.py

```diff
@@ -0,0 +1,62 @@
+"""
+Test api functions.
+"""
+
+from django.test.testcases import TestCase
+
+from django.contrib.auth.models import User
+
+from learningobjects.models import Course
+from learningobjects.api import create_course
+
+# pylint: disable=no-member
```

Comment:
We can try and fix them with the RC, something like:
```ini
[TYPECHECK]
generated-members=objects,_meta,id,etree
```
or whatever else you need.  It may not be worth it in the long run if we end up with a ton of these, but would help keep the code cleaner. I'm also mostly ok with just nuking that message altogether if it is too tedious to keep the list up